gradle/documentation/render-javadoc.gradle

```diff
@@ -278,6 +278,13 @@ configure(project(":solr:contrib:langid")) {
   }
 }
 
+configure(project(":solr:contrib:scripting-update-processor")) {
+  project.tasks.withType(RenderJavadocTask) {
+    // TODO: clean up split packages
+    javadocMissingIgnore = [ "org.apache.solr.update.processor" ]
```

Comment:
I would just add the javadoc placeholder instead...

Reply:
I think I understood what you mean, and backed out this change..   ;-)

Reply:
can we please change the package name and not create more NEW split package issues!

Reply:
Would moving this class to say `org.apache.solr.update.processor.scripting` avoid the split package issue?

Reply:
Now under a new `org.apache.solr.scripting.update` package to solve this.